modules/dnn/test/test_misc.cpp

```diff
@@ -63,6 +63,72 @@ TEST(imagesFromBlob, Regression)
     }
 }
 
+TEST(blobFromImageWithParams_4ch, NHWC_scalar_scale)
+{
+    Mat ch[4];
+    for(int i = 0; i < 4; i++)
+        ch[i] = Mat::ones(10, 10, CV_8U)*i;
+
+    std::vector<double> factorVec = {0.1, 0.2, 0.3, 0.4};
+
+    Scalar scalefactor(factorVec[0], factorVec[1], factorVec[2], factorVec[3]);
```

Comment:
You do not need  `std::vector<double>`, you can just initialize `Scalar scalefactor(0.1, 0.2, 0.3, 0.4);`

Reply:
the `factorVec[i]` will be used at the following code.